package.json

```diff
@@ -51,6 +51,7 @@
     "@types/node": "^7.0.4",
     "@types/run-sequence": "^0.0.28",
     "@types/rx": "2.5.33",
+    "autoprefixer": "^6.7.6",
```

Comment:
Removed all the autoprefixer stuff. So I can just rebase without conflicts if it's merged.